numpy/lib/utils.py

```diff
@@ -13,9 +13,86 @@
 __all__ = [
     'issubclass_', 'issubsctype', 'issubdtype', 'deprecate',
     'deprecate_with_doc', 'get_include', 'info', 'source', 'who',
-    'lookfor', 'byte_bounds', 'safe_eval'
+    'lookfor', 'byte_bounds', 'safe_eval', 'show_runtime'
     ]
 
+
+def show_runtime():
+    """
+    Print information about various resources in the system
+    including available intrinsic support and BLAS/LAPACK library
+    in use
+
+    See Also
+    --------
+    show_config : Show libraries in the system on which NumPy was built.
+
+    Notes
+    -----
+    1. Information is derived with the help of `threadpoolctl
+        <https://pypi.org/project/threadpoolctl/>`_ library.
```

Comment:
It seems sphinx is not rendering this: https://output.circle-artifacts.com/output/job/03b954b5-25ba-40b5-adfb-51afb9291f75/artifacts/0/doc/build/html/reference/generated/numpy.show_runtime.html

Will take a look